src/libs/ReportUtils.ts

```diff
@@ -3307,9 +3347,6 @@ function getIOUReportActionMessage(iouReportID: string, type: string, total: num
         case CONST.REPORT.ACTIONS.TYPE.APPROVED:
             iouMessage = `approved ${amount}`;
             break;
-        case CONST.REPORT.ACTIONS.TYPE.SUBMITTED:
```

Comment:
@FitseTLT  Please bring back this scope to improve readability as I mentioned before.

```javascript

case CONST.REPORT.ACTIONS.TYPE.SUBMITTED:
   iouMessage = getIOUSubmittedMessage(report);
   break;
```

Reply:
No for SUBMITTED we are not only setting `iouMessage` to then return an array with one fragment but returning an array of 4 message fragments unlike other cases. Please note the difference it has with other types.